### ISSUE TITLE
ngDialog modal closes on click

### ISSUE THREAD

**Issue by rvmishra** [likeastore/ngDialog#368]
I want to open a form in modal, where user can input few things and that stuff will be saved in backend. I came across this http://plnkr.co/edit/XDlzLtavZD3W2oWN1FQt which implements this thing.

But when i try to click anywhere to make an input, modal closes. I tried this in my local system as well - same behavior.

Tried following things but no result:
- I used ngDialog.openConfirm but it doesn't allow me to click anywhere(only tabs work.)
- Set closeByEscape to false - escape blocked but click makes modal disappear
- Set closeByDocument to false - clicks disabled and only tab allowed

Basically i need a modal:
- Which doesn't disappear on click and escape
- Allows tabs and clicks to make input
- Doesn't disappear on click in background
- Calls some function when submit is clicked
- Closes only when default close button is clicked

**Comment by PhiLhoSoft** [comment]
The plunk you link do doesn't work for me. It has the problem I point out in #373: instead of using `theme-plain`, you have to use `theme-default`.
And if I have:

        ngDialog.open({
            template: 'add-address.html',
            scope: $scope,
            closeByEscape: false,
            closeByDocument: false
        });

it seems to work as you want. Or do I miss something?
